packages/remix-dev/compiler/shims/cloudflare-pagesEntryModule.ts

```diff
@@ -0,0 +1,12 @@
+import { createPagesFunctionHandler } from "@remix-run/cloudflare-pages";
+import * as build from "@remix-run/server-build";
+
+const handleRequest = createPagesFunctionHandler({
+  build,
+  mode: process.env.NODE_ENV,
+  getLoadContext: context => context.env
```

Comment:
Let's add something to the docs about this. Maybe @kentcdodds can help us show people how to automatically access their durable objects in a loader or something.